.gitignore

```diff
@@ -22,7 +22,9 @@ nuget.exe
 *.ncrunchsolution
 *.*sdf
 *.ipch
-bower_components/
-node_modules/
 .vs/
 project.lock.json
+
+bower_components/
+node_modules/
+**/wwwroot/lib
```

Comment:
Is this the gist of the change in this pullrequest (removing wwwroot/lib from source control)?